target/linux/ath79/image/generic.mk

```diff
@@ -116,6 +115,16 @@ define Device/8dev_carambola2
 endef
 TARGET_DEVICES += 8dev_carambola2
 
+define Device/8dev_lima
+  SOC := qca9531
+  DEVICE_VENDOR := 8devices
+  DEVICE_MODEL := Lima
+  DEVICE_PACKAGES := kmod-usb2
+  IMAGE_SIZE := 15616k
```

Comment:
That doesn't match your partitioning at all ...

Reply:
Fixed to value 0xf40000 in dts file.